atm.js

```diff
@@ -1,21 +1,19 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance = 0;
+var savingsBalance = 0;
 
-}
+$("#c-depbtn").on("click", function(){
+  var userInput = $("#c-text").val();
+  checkingBalance += userInput;
+  $("#c-balance").html(checkingBalance);
+});
 
```

Comment:
i think you're pretty close here! Couple of notes. the `.val()` will always return a string. so if you were to add `"22" + 22` you would get `"2222"` That would make any bank very sad. I think what you want to do is parse it into an Integer first using `parseInt()`